src/jit/gentree.cpp

```diff
@@ -13044,6 +13116,39 @@ bool GenTree::DefinesLocalAddr(Compiler* comp, unsigned width, GenTreeLclVarComm
     return false;
 }
 
+//------------------------------------------------------------------------
+// IsLocalExpr: Determine if this is a LclVarCommon node and return some
+//              additional info about it in the two out parameters.
+//
+// Arguments:
+//    comp        - The Compiler instance
+//    pLclVarTree - An "out" argument that returns the local tree as a
+//                  LclVarCommon, if it is indeed local.
+//    pFldSeq     - An "out" argument that returns the value numbering field
+//                  sequence for the node, if any.
+//
+// Return Value:
+//    Returns true, and sets the out arguments accordingly, if this is
+//    a LclVarCommon node.
+
+bool GenTree::IsLocalExpr(Compiler* comp, GenTreeLclVarCommon** pLclVarTree, FieldSeqNode** pFldSeq)
+{
+    if (IsLocal())  // Note that this covers "GT_LCL_FLD." 
+    {
+        *pLclVarTree = this->AsLclVarCommon();
```

Comment:
Looking through the file there is a lot of inconsistency (and is not addressed, AFAICT, in the coding guidelines), so I have omitted the `this->` (though note that `IsLocalAddrExpr()` uses that form, and I have not changed it).